tf-modules/data-persistence/README.md

```diff
@@ -1,11 +1,14 @@
 # Data Persistence
 
-This module deploys data persistence resources, including DynamoDB tables and an Elasticsearch instance (which is optional).
+This module deploys data persistence resources, including DynamoDB tables, RDS database and tables, and an Elasticsearch instance (which is optional).
 
 ## Included resources
 
 - DynamoDB tables:
   - `AccessTokensTable`
+  - `SemaphoresTable`
+- RDS database within the PostgreSQL-compatible cluster
+- RDS database tables:
```

Comment:
These table names aren't correct for RDS. They should be:

- async_operations
- collections
- executions
- files
- granules
- granules_executions
- pdrs
- providers
- rules

Reply:
e41f234dd82f48333f3c14de8fa8710e9d93b81e